packages/perseus/src/perseus-api.tsx

```diff
@@ -168,6 +168,9 @@ export const ClassNames = {
         SELECTED: "perseus-radio-selected",
         OPTION_CONTENT: "perseus-radio-option-content",
     },
+    /**
+     * @deprecated This used to add a z-index of 3. Now it does nothing.
+     */
     INTERACTIVE: "perseus-interactive",
```

Comment:
I'm a little less familiar with this file, but would it be possible to remove this style now? If the style is doing nothing now, it seems like it could be a fairly safe thing to clean up! 

The only consumers of it are the widgets already in this PR, along with two more: 

- Input with Examples (A sub component of Numeric Input / Input Number that is used for desktop views) 
- Plotter

Reply:
This is in the API so I was hesitant to make any breaking changes.

I wasn't sure about the purpose of this enum. Could this be used as a selector? If so, maybe I should restore the class and just delete the styles.

Reply:
I'll explore uses of this enum, but it would be a breaking change, which would lead to a major version bump.

Reply:
I don't see any consumers of ClassNames so I think it's safe to remove it.

Reply:
I guess the Perseus API is internal? At least this was. Removed it.

Reply:
Excellent call, Sarah